src/components/WeatherCardComponent/styles.js

```diff
@@ -1,6 +1,6 @@
 import styled from "styled-components";
 
-export const WeatherCard = styled.div`
```

Comment:
Do we need this const?

Reply:
Yes, this is the main container of a card